### PR TITLE
feat(scheme): Add support for content://, file:// and base64:// on Android

### PR DESCRIPTION
**Pull request by WNemencha**

### CODE REVIEW

**Review by yunhanxia — APPROVED** [review]
Really need this merge

**Comment by biaji** [comment]
this pr should be merged asap.

**Comment by WNemencha** [comment]
@biaji @yunhanxia this repository seems to be dead, fork and merge it ;)
What we've been doing is to use [patch-package](https://www.npmjs.com/package/patch-package) so we don't have to wait for fix, nor to depend on a fork while having fix persisted between yarn installs

File: android/src/main/java/com/remobile/qrcodeLocalImage/RCTQRCodeLocalImage.java [hunk 1]
```diff
@@ -36,24 +43,42 @@ public void decode(String path, Callback callback) {
...
+        } else if (path.startsWith("base64://")) {
+            try {
+                byte[] decodedBytes = Base64.decode(path.replace("base64://", ""), 0);
+                scanBitmap = BitmapFactory.decodeByteArray(decodedBytes, 0, decodedBytes.length);
+            } catch (Exception e) {
+                e.printStackTrace();
+                scanBitmap = null;
+            }
         } else {
             scanBitmap = BitmapFactory.decodeFile(path, options);
```
Comment (iwashi1t) [inline android/src/main/java/com/remobile/qrcodeLocalImage/RCTQRCodeLocalImage.java:74]: this should be `scanBitmap = BitmapFactory.decodeFile(path.replace("file://", ""), options);`

**Review by ammarTariqSalsoft1447 — APPROVED** [review]

**Review by 452MJ — APPROVED** [review]

**Pull request closed without merging**